exporter/postgres/crunchy-postgres-exporter-pg13-el6.service

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
Why do we have el6 service file ? We are not building PG for el6.

Reply:
Removed PG13 el6 file